src/components/Table/Table.css

```diff
@@ -3,15 +3,14 @@
 .container {
   min-width: 100%;
   clear: both;
-  @media only screen and (max-width: 960px) {
-    margin: 0 -25px;
+  @media only screen and (max-width: 700px) {
```

Comment:
Leave the negative margin (unless you reduced the overall margin at small screen sizes) , we shouldn't have such large margins in mobile view